src/psm/README.md

```diff
@@ -69,6 +70,7 @@ check_power_grid
 | `-net` | Name of the net to analyze. Must be a power or ground net name. |
 | `-floorplanning` | Ignore non-fixed instances in the power grid, this is useful during floorplanning analysis when instances may not be properly placed. |
 | `-error_file` | File to write power grid errors to. |
+| `-require_terminals` | If specified, this will check of there are terminals on the net. |
```

Comment:
Do you think this should be off by default or on?

Reply:
I would suggest on by default, but that might have some short term implications on CI tests that run this command

Reply:
Then let's flip the option to on by default with an off option.  We might as well sort out the CI now as in another PR.